nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml

```diff
@@ -1,11 +1,13 @@
-# Requires the Asset component and the Twig bundle
-app.swagger_ui:
-    path: /api/doc
+# Expose your documentation as JSON swagger compliant
+app.swagger:
+    path: /api/doc.json
     methods: GET
-    defaults: { _controller: nelmio_api_doc.controller.swagger_ui }
+    defaults: { _controller: nelmio_api_doc.controller.swagger }
 
-# To expose your documentation as JSON
-#app.swagger:
-#    path: /api/doc.json
+## Requires the Asset component and the Twig bundle
+## $ composer require symfony/twig-bundle
+## $ composer require symfony/asset
```

Comment:
Can you replace these two lines with: `composer require twig asset`?

Reply:
done